docs/proposals/scheduling/20221227-node-resource-reservation.md

```diff
@@ -80,19 +82,44 @@ so for K8S, the 0-11 core is reserved and it can not be used.
 #### Story 2
 As a K8S cluster administrator, set the total amount of resources to be reserved directly via `node.annotation`, just like: `1C2Gi`.
 
+#### Story 3
+
+The cluster administrator just wants to reserve some cpu cores on the nods, and also wants to use reservation mechanism of kubelet to trim the allocatable resources.  For example, suppose the CPU capacity of node A is 32000m, the administrator reserve `0-3` CPU cores to the processes outside the Kubernetes, and kubelet reserve 8000m CPU to keep the `Node.Status.Allocatable` is 24000m.
```

Comment:
If kubelet.reserved < node.anno.reserved, don't need to change `nodeinfo.alloc` as well?

Reply:
Yes.

Reply:
ok, awesome